### DISCUSSION TITLE
Hosting game files on GitHub

### DISCUSSION THREAD

**Discussion by QuakeEye** [community/community#22470] [New to GitHub]
<p>Hey there!</p>
<p>I was wondering if it is allowed to host game files on github and have a launcher I created automatically download them to update the game?</p>
<p>Thanks in advance!</p>
<p>(I am asking this since I could not find info on this subject, neither did I find anything relevant on it in the TOS)</p>

**Comment by jfullstackdev** [comment]
<p>what do you mean by that ? you want to put game files on GitHub, for others to download ?</p>

**Comment by QuakeEye** [comment]
<p>Yes that is exactly it, only it would be auto downloaded by my launcher instead of people manually downloading the files</p>

**Comment by jfullstackdev** [comment, marked as answer]
<aside class="quote no-group" data-username="GuidoMintjes" data-post="3" data-topic="234452">
<div class="title">
<div class="quote-controls"></div>
 GuidoMintjes:</div>
<blockquote>
<p>only it would be auto downloaded by my launcher</p>
</blockquote>
</aside>
<p>yup, GitHub is being used to deploy apps, unless that app is malicious or violates anything of the GitHub rules and software in general</p>
